test/unit/iwdp-specs.js

```diff
@@ -0,0 +1,82 @@
+// transpile:mocha
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import IWDP from '../../lib/iwdp';
+import { SubProcess } from 'teen_process';
+import request from 'request-promise';
+import Promise from 'bluebird';
```

Comment:
We have generally been using `B` as the import for `bluebird`, mostly to distinguish it from the vanilla `Promise` implementation, which has different capabilities.